jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -259,6 +260,12 @@ static final void initializeClassLoaders() {
 		
 		/*[IF Sidecar19-SE]*/
 		jdk.internal.misc.VM.initLevel(1);
+		/* 
+		 * Keep following code as first call of this method which ensures that the field jdk.internal.reflect.langReflectAccess 
```

Comment:
I think the wording of this comment needs to be changed. It is no longer the first call of this method. The testing passed, so once that it done I will merged this and the extension changes.

Reply:
Good catch! the comments are updated.